src/main/resources/static/css/style.css

```diff
@@ -60,4 +59,13 @@ h1 {
 }
 .login {
     --bs-btn-hover-bg: 0.3s;
-}
\ No newline at end of file
+}
+
+.email {
+    margin-top: 50px ;
+}
```

Comment:
you really like to add empty lines in the java code where they do not belong but do not put them in css, where they should be so stuff remains readable ^^